src/bindings/PyDP/algorithms/distributions.cpp

```diff
@@ -6,31 +6,31 @@
 
 using namespace std;
 namespace py = pybind11;
-namespace dpi = differential_privacy::internal;
+namespace dpinternal = differential_privacy::internal;
```

Comment:
I'm not against this, but if we go the route of only shortening differential_privacy namespace, I would completely leave behind renaming subnamespaces and just use dp::X where necessary.

Reply:
IMO
If we just stick to one namespace in a given file, we can shorten the namespace. 
But in case multiple namespaces are in the files, maybe stick to writing dp::X?

Reply:
IMO
If we just stick to one namespace in a given file, we can shorten the namespace. 
But in case multiple namespaces are in the files, maybe stick to writing dp::X?

Reply:
hey @alejandrosame taken up your idea and made the changes in the latest commit.

Reply:
Ok! In any case we can always revise it later, if we find out that it interferes with readability for more complex bindings.